playground/terraform/infrastructure/variables.tf

```diff
@@ -94,12 +94,12 @@ variable "redis_name" {
 
 variable "redis_tier" {
   description = "Tier of Redis"
-  default     = "STANDARD_HA"
```

Comment:
Why are we downgrading?

Reply:
Hello, @damondouglas. Even on current Production we're using BASIC Redis version and it's enough

Reply:
We don't downgrade. Currently we use Basic Redis instance. According to monitoring, we don't utilize it enough to switch to Standard instance.

Reply:
@rshamunov What if the usage grows? What is the plan to monitor and respond to capacity issues?

Reply:
@damondouglas, currently Playground use Redis for tasks that don't utilize even Basic tier capacity. We are not at risk of performance issues, even if usage of Playground environment grows significantly. Besides, we can reconfigure Playground to use new Redis instance if we need.